bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/sensors/PresenceHandler.java

```diff
@@ -39,10 +45,38 @@
 public class PresenceHandler extends HueSensorHandler {
     public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_PRESENCE_SENSOR);
 
+    private final Logger logger = LoggerFactory.getLogger(PresenceHandler.class);
+
     public PresenceHandler(Thing thing) {
         super(thing);
     }
 
+    @Override
+    public void handleCommand(String channel, Command command) {
+        HueClient hueBridge = getHueClient();
+        if (hueBridge == null) {
+            logger.warn("hue bridge handler not found. Cannot handle command without bridge.");
+            return;
+        }
+
+        final FullSensor sensor = lastFullSensor;
+        if (sensor == null) {
+            logger.debug("hue sensor not known on bridge. Cannot handle command.");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.conf-error-wrong-sensor-id");
+            return;
+        }
+
+        SensorConfigUpdate configUpdate = new SensorConfigUpdate();
+        switch (channel) {
+            case CHANNEL_ENABLED:
+                configUpdate.setOn(OnOffType.ON.equals(command));
+                break;
+        }
```

Comment:
For other channels, I think you have to call the parent class (which handle few other channels).

Reply:
On the first glance, yes, but currently the presence sensor only has read-only channels thus no command has be handled except this one.